lib/net/common.js

```diff
@@ -166,10 +167,7 @@ exports.BAN_SCORE = 100;
  */
 
 exports.nonce = function nonce() {
-  const data = Buffer.allocUnsafe(8);
-  data.writeUInt32LE((Math.random() * 0x100000000) >>> 0, 0, true);
-  data.writeUInt32LE((Math.random() * 0x100000000) >>> 0, 4, true);
-  return data;
+  return random.randomBytes(8);
```

Comment:
Looks like this one is only used in ping/pong challenges to peers, and BIP152 (compact blocks) which require a 64-bit nonce to prevent collision attacks on short ids.